source/blog/2019-01-07-ember-3-7.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Ember 3.7 Released
+author: Melanie Sumner, Kenneth Larsen
+tags: Releases, 2019, 3, 3.7
+responsive: true
+---
+
+Today the Ember project is releasing 3.7 version 3.7 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.8 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.7
+Ember.js 3.7 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is zero (0) new feature, zero (0) deprecations, and three (3) bugfixes in this 3.7 version.
+
+Note that in Ember.js 3.7 support for Node 4 has been explicitly dropped. This means that if you are on Node 4 and want to upgrade you will need to upgrade your version of Node first.
```

Comment:
we might want to mention here that Node 4 is no longer supported by the Node.js team either, and that Ember CLI had already dropped support in early 2018.

also, Node 6 support will end in the next few months, so maybe we should start to nudge people towards upgrading if they still use that.